ckanext/privatedatasets/actions.py

```diff
@@ -114,11 +205,11 @@ def package_acquired(context, request_data):
                         context_pkg_update['user'] = user.get('name', '')
 
                         plugins.toolkit.get_action('package_update')(context_pkg_update, dataset)
-                        log.info('Allowed Users added correctly')
+                        log.info('Action -%s access to user- ended successfully' % context['method'])
```

Comment:
The log message you are generating is something like this: "Action -revoke access to user- ended successfully". I do not understand why you are using those dashes. I would prefer something like: "Action revoke access to dataset ended successfully" or simply "Revoke access to dataset ended successfully". 

PS: It's "access to **dataset**" not "access to **user**"